src/librustdoc/html/render/mod.rs

```diff
@@ -1745,12 +1745,17 @@ fn print_sidebar(cx: &Context<'_>, it: &clean::Item, buffer: &mut Buffer) {
         ty = it.type_(),
         path = relpath
     );
+
     if parentlen == 0 {
-        // There is no sidebar-items.js beyond the crate root path
-        // FIXME maybe dynamic crate loading can be merged here
+        write!(
+            buffer,
+            "<script defer src=\"{}sidebar-items{}.js\"></script>",
+            relpath, cx.shared.resource_suffix
+        );
     } else {
-        write!(buffer, "<script defer src=\"{path}sidebar-items.js\"></script>", path = relpath);
+        write!(buffer, "<script defer src=\"{}sidebar-items.js\"></script>", relpath);
```

Comment:
There is no need for this `else`. The `else` block is the same as the `if` but invalid: it needs to use `resource_suffix`.

Reply:
No, the else block is loading the per-module one which is unsuffixed.

Reply:
Ah right, because this approach is like this... Ok, I think globally, this is just the wrong approach, I'll comment about it.